### DISCUSSION TITLE
How to get root key in codespaces

### DISCUSSION THREAD

**Discussion by DragonX1901** [community/community#137380] [Codespaces]
I was working on codespaces and it asked me for the root password. How can I get the root password?

**Comment by chengwy2014** [comment]
U can use “sudo -i” to use root cmds. Neednt pwd.
Have a nice day :)

**Comment by chengwy2014** [comment]
Plz close your discussions if you know it. 
If still have something problem on it, u can find me or give me e-mail: readingcheng@outlook.com
